etl_framework/config_mixins/FiltersMixin.py

```diff
@@ -29,6 +30,26 @@ def add_filters(self, filter_mappings):
             self.set_filter(getattr(filter_mappings, self.get_filter()))
             self.set_post_filter(getattr(filter_mappings, self.get_post_filter()))
 
+    def add_filters_from_module(self, filter_module):
```

Comment:
@ianlofs `add_filters` doesn't need the if-else block anymore.  Just assume that input is a dictionary.  We'll have 2 ways to add filters now.  `add_filters` and `add_filters_from_module`.  Other than that, looks good 👍